API.md

```diff
@@ -0,0 +1,387 @@
+# FBInk Command Line App Manual
+
+## Synopsis
+
+* ```sh
+  fbink [OPTIONS] [STRING ...]
+  ```
+
+  Print `STRING`s on your device's screen.
+
+* ```sh
+  fbink [-WHf] [-k] --refresh [top=NUM,left=NUM,width=NUM,height=NUM]
```

Comment:
A bit wary of mentioning `-k, --cls` here, especially since it's detailed below, and it makes the explanation below a lie, because it *will* modify the framebuffer ;).

It's also missing a whole bunch of supported flags.

Reply:
GitHub shows this weirdly. Are you referring to a definition in the Synopsis? Which one?

Reply:
Could you please review this raw Markdown for a sec?

The description may not be lining up to the correct example line that you see. This is unambiguous when you read the code in the matrix/Markdown.

Reply:
The associated description is below the `---sh` block

Reply:
I am actually reading the raw, haven't even looked at how it looks like processed ;).

Reply:
And, yeah, my bad, that one is *not* missing flags ;).

https://github.com/NiLuJe/FBInk/blob/94305d2d2b40983dee12b2a236dac573ebbd556f/fbink_cmd.c#L273

Reply:
removed this "todo". 2b471773b12672fd198d9f31e2860eba8a39e7c2